### PR TITLE
ci: add `tuiTab` for cypress testing

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/demo-integrations/cypress/support/commands.ts [hunk 1]
```diff
@@ -18,7 +19,7 @@ declare global {
             tuiSetNightMode: typeof tuiSetNightMode;
             tuiHideNavigation: typeof tuiHideNavigation;
 
-            tab(options?: Partial<{shift: boolean}>): Chainable;
+            tuiTab(): Chainable;
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/commands.ts:22]:
```suggestion
            tuiTab(direction: 'forward' | 'backward'): Chainable;
```

File: projects/demo-integrations/cypress/support/type-tab.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    return (
+        (nodeName === 'a' ||
+            !!$element.attr('tabindex') ||
+            (DISABLEMENT_ELEMENTS.includes(nodeName) && $element.is(':enabled'))) &&
+        $element.is(':visible')
+    );
+}
+
+export function tuiTab($subject: Cypress.PrevSubject, direction = 'forward'): void {
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/type-tab.ts:77]:
```suggestion
export function tuiTab($subject: Cypress.PrevSubjectMap<void>[Cypress.PrevSubject], direction = 'forward'): void {
```

Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/type-tab.ts:77]:
```suggestion
export function tuiTab($subject: Cypress.PrevSubject, direction: 'forward' | 'backward' = 'forward'): void {
```

File: projects/demo-integrations/cypress/support/type-tab.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    const thenable: Cypress.Chainable = $subject
+        ? cy.wrap($subject, {log: false})
+        : cy.focused({log: false});
+
+    thenable.then($el => nextTabbable($el, direction)).focus({log: false});
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/type-tab.ts:82]: add `return` to make this command `chainable`

Comment (nsbarsukov) [reply projects/demo-integrations/cypress/support/type-tab.ts:82]: and make return type of the function equal to `Cypress.Chainable`

File: projects/demo-integrations/cypress/support/commands.ts [hunk 4]
```diff
@@ -33,6 +34,9 @@ Cypress.Commands.add('tuiWaitKitDialog', tuiWaitKitDialog);
 Cypress.Commands.add('tuiSetNightMode', tuiSetNightMode);
 Cypress.Commands.add('tuiHideNavigation', tuiHideNavigation);
 
+// @ts-ignore
+Cypress.Commands.add('tuiTab', tuiTab);
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/commands.ts:38]:
```suggestion
Cypress.Commands.add('tuiTab', {prevSubject: ['optional', 'element', 'window', 'document']}, tuiTab);
```

Comment (nsbarsukov) [reply projects/demo-integrations/cypress/support/commands.ts:38]: remove `@ts-ignore` after resolving my another comments

File: projects/demo-integrations/cypress/support/type-tab.ts (hunk 2 above)
Comment (nsbarsukov) [reply projects/demo-integrations/cypress/support/type-tab.ts:77]: it is just a string for now

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/demo-integrations/cypress/support/commands.ts (hunk 1 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/commands.ts]: fixed

File: projects/demo-integrations/cypress/support/type-tab.ts (hunk 2 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/type-tab.ts]: fixed

File: projects/demo-integrations/cypress/support/type-tab.ts (hunk 3 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/type-tab.ts]: fixed

File: projects/demo-integrations/cypress/support/type-tab.ts (hunk 2 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/type-tab.ts]: fixed

File: projects/demo-integrations/cypress/support/commands.ts (hunk 4 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/commands.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Comment by splincode** [comment]
@waterplea @nsbarsukov please, review me)

**Review by nsbarsukov — APPROVED** [review]

**Comment by splincode** [comment]
@vladimirpotekhin hello, please review me)

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**